index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="./assets/css/main.css" />
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <div class="section">
+      <p class="quote">
```

Comment:
I suggest to use more semantic tags for quote. This article can be helpful or you can search more about that.
https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/

Reply:
I repleced it with q tag.